Gini-iOS-SDK/GINIDocumentTaskManager.h

```diff
@@ -78,6 +79,21 @@
 - (BFTask *)createDocumentWithFilename:(NSString *)fileName
                              fromImage:(UIImage *)image __attribute__((deprecated("use createPartialDocumentWithFilename:fromData:docType:cancellationToken: method instead")));
 
+/**
+ * Creates a new document from the given image.
+ *
+ * @param fileName      The file name of the document.
+ * @param image         An image representing the document.
+ * @param metadata      (Optional) The document metadata.
+ *
+ * @returns             A `BFTask*` that will resolve to a `GINIDocument` instance representing the created document.
+ *                      Please notice that it is very unlikely that the created document is already fully processed, so
+ *                      the extractions may not yet exist.
+ */
+- (BFTask *)createDocumentWithFilename:(NSString *)fileName
+                             fromImage:(UIImage *)image
+                              metadata:(GINIDocumentMetadata *)metadata __attribute__((deprecated("use createPartialDocumentWithFilename:fromData:docType:cancellationToken: method instead")));
```

Comment:
Adding a method that's immediately deprecated? Idk...

Reply:
I thought about this and I think for now it's the best solution, since some clients might not want to use PartialI/Composite (although is not recommended). In the future they will disappear